README.md

```diff
@@ -83,6 +83,19 @@ of options:
  * [Libvirt][libvirt]
  * [Cockpit][cockpit]
 
+## Submitting patches
+
+When sending patches to the project, the submitter is required to certify that
+they have the legal right to submit the code. This is achieved by adding a line
+
+    Signed-off-by: Real Name <email@address.com>
```

Comment:
I understand that this is `explicit` and of course you  can still force contributor to sign DCOs, although your license is already covering patents. The question is, if it is necessary, or really improves things and if you know of some sources for that.
 
For instance github points to this [legal site](https://opensource.guide/legal/#does-my-project-need-an-additional-contributor-agreement). It is stated there that you only need it when your license does not cover it, or if you want a lightweight solution, in case your lawyers insist that you need that (although it is not necessary in their eyes, when your license covers patents). Further, for instance [Bradley M. Kuhn from conservancy](https://sfconservancy.org/blog/2014/jun/09/do-not-need-cla/) wrote that it might be nice to have, but that most of there projects don't use it and that is perfectly fine.

I am sure we can live with that little inconvenience, of doing a `-s`, when there are good reasons for it. But let's double check, if this is needed right now, or needed at all.